frontend/lib/src/components/widgets/FileUploader/FileDropzoneInstructions.test.tsx

```diff
@@ -30,34 +31,33 @@ const getProps = (props: Partial<Props> = {}): Props => ({
 describe("FileDropzoneInstructions widget", () => {
   it("renders without crashing", () => {
     const props = getProps()
-    const wrapper = shallow(<FileDropzoneInstructions {...props} />)
+    render(<FileDropzoneInstructions {...props} />)
 
-    expect(wrapper).toBeDefined()
+    expect(
+      screen.getByTestId("stFileDropzoneInstructions")
+    ).toBeInTheDocument()
   })
 
   it("shows file size limit", () => {
     const props = getProps({ maxSizeBytes: 2000 })
-    const wrapper = mount(<FileDropzoneInstructions {...props} />)
-    const smallWrapper = wrapper.find(Small)
+    render(<FileDropzoneInstructions {...props} />)
 
-    expect(smallWrapper.text()).toBe("Limit 2KB per file")
+    expect(screen.getByText("Limit 2KB per file")).toBeInTheDocument()
   })
 
   it("renders without extensions", () => {
     const props = getProps({
       acceptedExtensions: [],
     })
-    const wrapper = mount(<FileDropzoneInstructions {...props} />)
-    const smallWrapper = wrapper.find(Small)
-    expect(smallWrapper.text()).toMatch(/per file$/)
+    render(<FileDropzoneInstructions {...props} />)
+    expect(screen.getByText(/per file$/)).toBeInTheDocument()
   })
 
   it("renders with extensions", () => {
     const props = getProps({
       acceptedExtensions: ["jpg"],
     })
-    const wrapper = mount(<FileDropzoneInstructions {...props} />)
-    const smallWrapper = wrapper.find(Small)
-    expect(smallWrapper.text()).toMatch(/•/)
+    render(<FileDropzoneInstructions {...props} />)
+    expect(screen.getByText(/•/)).toBeInTheDocument()
```

Comment:
Nit: I know this was pulled from the previous test, but would be nice to check w/ `screen.getByText(/• JPG/)`